doc/contributing.rst

```diff
@@ -1,9 +1,10 @@
-.. default-role:: code
-
 ============
 Contributing
 ============
 
+.. default-role:: code
+.. include:: rstcommon.rst
```

Comment:
In that include file we have `.. default-role:: nim`. Do we need `.. default-role:: code` above it too?

Reply:
For `nim rst2html` we don't need `.. default-role:: code`. This is for compatibility with GitHub. Github doesn't seem to include files and it does not highlight Nim even if inserted directly to file (yet?). So effectively it sees only `.. default-role:: code`, which is satisfactory.

This trick was proposed by @timotheecour , see discussion around https://github.com/nim-lang/Nim/pull/17585#issuecomment-810630045